projectroles/views_ajax.py

```diff
@@ -368,9 +368,21 @@ def post(self, request, *args, **kwargs):
         project = self.get_project()
         user = request.user
         timeline = get_backend_api('timeline_backend')
-        tag_state = get_tag_state(project, user)
+        tag_state = app_settings.get(
```

Comment:
The concept of "tag" will no longer exist after this, so the variable should also be renamed. Otherwise it might be a bit confusing :)